final_task/rss_reader/MyException.py

```diff
@@ -0,0 +1,3 @@
+class MyException(Exception):
```

Comment:
Please avoid using names such as `my...` in the production code. This exception class name should briefly describe the situation when you want to raise it.